SimpleWAWS/Code/CsmExtensions/CsmManager.cs

```diff
@@ -159,7 +172,12 @@ static CsmManager()
             await response.EnsureSuccessStatusCodeWithFullError();
 
             var result = await response.Content.ReadAsAsync<CsmSubscriptionsArray>();
-            return result.value.ToDictionary(k => k.displayName, v => v.subscriptionId);
+
+            response = await jenkinsClient.HttpInvoke(HttpMethod.Get, ArmUriTemplates.Subscriptions.Bind(""));
+            await response.EnsureSuccessStatusCodeWithFullError();
+
+            var result2 = await response.Content.ReadAsAsync<CsmSubscriptionsArray>();
+            return result.value.Union(result2.value).ToDictionary(k => k.displayName, v => v.subscriptionId);
```

Comment:
this could result in duplicate keys exception. Subscription names are not globally unique.